Lib/fontTools/agl.py

```diff
@@ -1,8 +1,12 @@
+# -*- coding: utf-8 -*-
 # The table below is taken from
 # http://www.adobe.com/devnet/opentype/archives/aglfn.txt
 
-from __future__ import print_function, division, absolute_import
+from __future__ import (print_function, division, absolute_import,
+                        unicode_literals)
```

Comment:
yes, it's not good to mix the two. 
I think it's best to keep the `unicode_literals`, but make sure the native `str` type is used in the `AGL2UV` and `UV2AGL` dictionaries.

See d8c4058